doc/ref/configuration/minion.rst

```diff
@@ -307,6 +307,21 @@ Set to zero if the minion should shutdown and not retry.
 
     retry_dns: 30
 
+.. conf_minion:: retry_dns
+
+``retry_dns_count``
+-------------
```

Comment:
I know this is picky, but the number of dashes needs to match the line above it or the doc build will have a warning. (We need to figure out a way to get that to bubble up in the PR tests still.)

Can you add a `versionadded` tag in here as well?